src/challenge-status/entities/challenge-status.entity.ts

```diff
@@ -0,0 +1,23 @@
+import { Entity, Column, Timestamp } from 'typeorm';
+import { InputType, Field, ObjectType, ArgsType } from '@nestjs/graphql';
+import { State } from './state.enum';
+
+@Entity('ChallengeStatus')
+@ObjectType('ChallengeStatus')
+export class ChallengeStatusEntity {
```

Comment:
Missing `studentId` and `challengeId`, both references to ID of corresponding entity.